gensim/test/test_phrases.py

```diff
@@ -8,17 +8,29 @@
 """
 
 
+import contextlib
 import logging
-import unittest
 import os
-import sys
+import shutil
+import tempfile
+import unittest
+
+import six
 
 from gensim import utils
-from gensim.models.phrases import SentenceAnalyzer, Phrases, Phraser, pseudocorpus
+from gensim.models.phrases import SentenceAnalyzer, Phrases, Phraser
+from gensim.models.phrases import pseudocorpus, original_scorer
 from gensim.test.utils import common_texts
 
-if sys.version_info[0] >= 3:
-    unicode = str
+
+@contextlib.contextmanager
```

Comment:
This looks useful, please move it to `gensim.test.utils` https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/utils.py